spec.emu

```diff
@@ -1131,6 +1131,19 @@ location: https://github.com/tc39/proposal-decimal/
     </emu-note>
   </emu-clause>
 
+  <emu-clause id="sec-decimal128.prototype.tojson">
+    <h1>Decimal128.prototype.toJSON ( )</h1>
+    <p>It performs the following steps when called:</p>
+    <emu-alg>
+      1. Let _O_ be the *this* value.
+      1. Perform ? RequireInternalSlot(_O_, [[Decimal128Data]]).
```

Comment:
Curious, if we are going to throw a TypeError anyway. Is this line observable?

Reply:
I killed those two lines. Good spot. We do the same with `valueOf` (no brand checks, just throw).